src/components/map/mapbox-style.ts

```diff
@@ -31,7 +31,7 @@ function createMapboxStreetsV6Style() {
             'https://unpkg.com/@mapbox/maki@4.0.0/icons/' +
             iconName +
             '-15.svg',
-          // TODO(volfpeter): Is this correct (copied from views.map.styles.mapbox)?
+          // TODO(vp): Is this correct (copied from views.map.styles.mapbox)?
           // Seems like the imgSize prop does not exist.
```

Comment:
`imgSize` was removed in OpenLayers 8 so we can get rid of this as well. See more details [here](https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md#removal-of-olstyleicons-imgsize-property).